UnityProject/Assets/Scripts/Items/PlaceableTile.cs

```diff
@@ -140,5 +136,9 @@ private class PlaceableTileEntry
 		[Tooltip("The amount of this item required to place tile.")]
 		[SerializeField]
 		public int itemCost = 1;
+
+		[Tooltip("Time it takes to place the tile")]
+		[SerializeField]
+		public float timeToPlace = 1f;
```

Comment:
Why public?

Reply:
![image](https://user-images.githubusercontent.com/56727168/122653800-162b1880-d13f-11eb-8e03-e6b690ebfcf8.png)

Reply:
stuff needs to access it

Reply:
yeah, I didn't see the inner class in my phone. Also was just awakening e-e